magefiles/magefile.go

```diff
@@ -222,10 +221,18 @@ func (ci CI) TestE2E() error {
 		return fmt.Errorf("error when bootstrapping cluster: %v", err)
 	}
 
+	if err := registerPacServer(); err != nil {
```

Comment:
As mentioned in my other comment, I have some concerns about the stability of the sprayproxy host. Could we  add a retry logic here? That way we could avoid test failures caused by a temporary downtimes of sprayproxy host.
```suggestion
	if err := retry(registerPacServer, 3, 10*time.Second); err != nil {
```

Reply:
wdyt about adding a retry logic